src/main/java/org/broadinstitute/hellbender/dev/tools/walkers/bqsr/DF_BaseRecalibrator.java

```diff
@@ -256,7 +355,22 @@ public void processElement(ProcessContext c) throws Exception {
                 }));
     }
 
-    private static <T> void saveSingleResult(final PCollection<T> collection, String fname) {
+    // The collection's expected to contain a single object.
+    // serializeSingleObject will serialize it to the given file name (either local or GCS).
+    // returns true if we're saving to GCS.
+    // Of course if you run on the cloud and specify a local path, the file will be saved
+    // on a cloud worker, which may not be very useful.
+    private static <T> boolean serializeSingleObject(PCollection<T> collection, String fname) {
```

Comment:
Can this (and the helper methods below) be extracted into a general-purpose utility method in `utils/dataflow/DataflowUtils`?